Update Flutter Docs to reference Swift SDK methods (#4759)
braze-inc/braze-docs: _docs/_developer_guide/platform_integration_guides/flutter/flutter_sdk_integration.md

{% tab iOS %}
{% subtabs global %}
{% subtab SWIFT %}
Add Braze SDK import at the top of the `AppDelegate.swift` file:
```swift
import BrazeKit
```

> In the same file, create the Braze configuration object with the API key and endpoint of your Braze app within the `application:didFinishLaunchingWithOptions` method and replace the API identifier key and endpoint with your value. Create the Braze instance using the configuration, and create a braze static property on AppDelegate for easy access to SDK features:

Reviewer: Tweaked a bit and removed some words:
```suggestion
In the same file, create the Braze configuration object in the `application:didFinishLaunchingWithOptions` method and replace the API key and endpoint with your app's values. Then, create the Braze instance using the configuration, and create a static property on the AppDelegate for easy access:
```

```swift
static var braze: Braze? = nil

func application(
  _ application: UIApplication,
  didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil
) -> Bool {
  let configuration = Braze.Configuration(
    apiKey: "<BRAZE_API_KEY>",
    endpoint: "<BRAZE_ENDPOINT>"
  )
  // Enable logging of general SDK information (e.g. user changes, etc.)

>   configuration.logger.level = .info

Reviewer 2: Imo we can remove logger related lines for simplicity + less maintenance burden (same for obj-c example). They can visit the sample app or iOS docs to see something more fleshed out

  let braze = Braze(configuration: configuration)
  AppDelegate.braze = braze

  return true
}
```
{% endsubtab %}
{% subtab OBJECTIVE-C %}

> Add Braze SDK import at the top of the `AppDelegate.m` file:

Reviewer: ```suggestion
Import `BrazeKit` at the top of the `AppDelegate.m` file:
```

```objc
@import BrazeKit;
```

> In the same file, create the Braze configuration object with the API key and endpoint of your Braze app within the `application:didFinishLaunchingWithOptions` method and replace the API identifier key and endpoint with your value. Create the Braze instance using the configuration, and create a braze static property on AppDelegate for easy access to SDK features:

Reviewer: ```suggestion
In the same file, create the Braze configuration object in the `application:didFinishLaunchingWithOptions` method and replace the API key and endpoint with your app's values. Then, create the Braze instance using the configuration, and create a static property on the AppDelegate for easy access:
```
